packages/jellyfish-api-core/src/category/icxorderbook.ts

```diff
@@ -280,3 +344,73 @@ export interface ICXListOrderOptions {
   /**  Display closed orders (default: false) */
   closed?: boolean
 }
+
+/** ICX listHTLC options */
+export interface ICXListHTLCOptions {
+  /** Offer txid  for which to list all HTLCS */
+  offerTx?: string
+  /** Maximum number of orders to return (default: 20) */
+  limit?: number
+  /** Display refunded HTLC (default: false) */
+  refunded?: boolean
+  /** Display claimed HTLCs (default: false) NOTE(surangap): in c++ side desciption this is mentioned as "claimed". should be corrected */
+  closed?: boolean
+}
+
+/** ICX claimed DFCHTLC info */
+export interface ICXClaimDFCHTLCInfo {
```

Comment:
hmm.. `ICXClaimDFCHTLCInfo`, `ICXEXTHTLCInfo`, `ICXEXTHTLCInfo` are not updated in doc...
are these using somewhere??

Reply:
`ExtHTLC ` will be added in next PR `surangap/icx-submitexthtlc`